mirar/utils/send_email.py

```diff
@@ -25,8 +25,8 @@ def send_gmail(
     email_recipients: str | list[str],
     email_subject: str,
     email_text: str,
-    email_sender: str = os.getenv("WATCHDOG_EMAIL"),
-    email_password: str = os.getenv("WATCHDOG_EMAIL_PASSWORD"),
+    email_sender: str = os.environ.get("WATCHDOG_EMAIL"),
```

Comment:
What is the purpose of this?

Reply:
(And the line below)

Reply:
Pipeline uses both os.getenv and os.environ.get in various files. Should be standardised.
I think when I did this I choose os.environ.get since [getenv uses environ](https://docs.python.org/3/library/os.html#os.getenv) and supposedly there are issues with putenv.

I then failed to follow this rule myself.

Reply:
I've now updated this so all calls to this use os.environ.get

Reply:
Can you please just use `os.getenv` everywhere? I think that is what I was mostly doing before, and it's shorter.

Reply:
At least as far as I understood it there is no downside to using getenv, and nowhere do we ever use putenv. Happy to discuss dorectly if you have opinions on that though.

Reply:
Okay, done.